### DISCUSSION TITLE
New roms not showing up in 86box

### DISCUSSION THREAD

**Discussion by Tomeister1** [86Box/86Box#3958] [Q&A] [answered]
Hi I am running a windows 11 laptop and am trying to access the new recently added roms I have downloaded them and copied into the roms folder but so far I have not been able to access them can anyone explain what could be the issue? Please let me know thanks.

**Comment by lemondrops** [comment, marked as answer]
If by "new roms" you mean https://github.com/86Box/roms/commit/a44c67c77de949878428df643236f18eb80f2ab9, then the build supporting them isn't ready yet, wait for build 5307 to complete.
